Added sprint references, fixed some lilnks and cleaned up outdated info (#762)
EuroPython/website: src/content/pages/faq.mdx

## **Q. What support do you give to speakers?**

> A. Accepted speakers are eligible for a free ticket. Detailed information regarding the types of tickets available to speakers has been sent directly via email. In addition, speakers are encouraged to apply for additional [Financial Aid](/finaid), should there be a need.

Reviewer: ```suggestion
A. Accepted speakers are eligible for a free ticket. Detailed information regarding the types of tickets available to speakers has been sent directly via email. In addition, speakers were encouraged to apply for additional [Financial Aid](/finaid), if needed.
```

Reviewer: finaid is finished I think

## **Q. When can sponsors start signing up?**

A. We are open for sponsorship signup, though availability for some packages may be limited or pending. You can find all the details of the 2024 sponsorship packages and options on our [sponsorship page](/sponsor), as well as the demographics of previous years.

> **Sponsors who sign up by 15th of April again received a 10% discount.**

Reviewer: ```suggestion
**Sponsors who signed up by 15th of April again received a 10% discount.**
```
